sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,11 +1,16 @@
+# 2019-09-23 1.1.0
+
+- Add browser support. Authentication with `@azure/identity` is not supported yet - use a connection string instead.
```

Comment:
Users of this library are not aware of what `@azure/identity` is. If they have been using AAD, then that would be by using the `@azure/ms-rest-nodeauth` library.

```suggestion
- Add browser support. Authentication using Azure Active Directory credentials is not supported yet - use a connection string instead.
```